vector_20_change_sign_1.py

```diff
@@ -0,0 +1,19 @@
+# importing numpy
+import numpy as np
+
+
+def change_sign_vector(np1, range1, range2):
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/